addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -59,7 +60,7 @@ def test_efaktur_csv_output_1(self):
             _csv_row(OF_HEAD_LIST, ','),
         )
         # remaining lines
-        line_4 = '"FK","01","0","0000000000001","5","2019","1/5/2019","12345","l10ntest","","100","10","0","","0","110","0","0","INV/2019/00001 12345","0"\n'
+        line_4 = '"FK","01","0","0000000000001","5","2019","1/5/2019","000000000000000","12345#NIK#NAMA#l10ntest","","100","10","0","","0","110","0","0","INV/2019/00001 ","0"\n'
```

Comment:
not great that there is a " " at the end of the ref when the nik isn't used :thinking:

Reply:
also I don't see a test where vat is not set